kitops/modelkit/kitfile.py

```diff
@@ -189,167 +152,35 @@ def load(self, path):
 
         # try to load the kitfile
         try:
-            data = yaml.safe_load(kitfile_path.read_text(encoding="utf-8"))
-
+            data: dict = yaml.safe_load(kitfile_path.read_text(encoding="utf-8"))
         except yaml.YAMLError as e:
             if mark := getattr(e, "problem_mark", None):
                 raise yaml.YAMLError(
-                    f"Error parsing Kitfile at line{mark.line + 1}, column:{mark.column + 1}."
+                    "Error parsing Kitfile at " + f"line{mark.line + 1}, " + f"column:{mark.column + 1}."
                 ) from e
             else:
                 raise
 
-        try:
-            validate_dict(value=data, allowed_keys=self._kitfile_allowed_keys)
-        except ValueError as e:
-            raise ValueError(
-                f"Kitfile must be a dictionary with allowed keys: {', '.join(self._kitfile_allowed_keys)}"
-            ) from e
+        if any(ALLOWED_KEYS.difference(data.keys())):
```

Comment:
This will cause issues if we add a new field right?

Reply:
Yes. It is only a reflection of the existing code.

https://github.com/jozu-ai/pykitops/blob/82111eedad57f356c8a88c9d4f4385d64ed66fef/kitops/modelkit/kitfile.py#L202-L207

Reply:
Let me ponder this. I am not sure we care if we are making the classes now.